lib/rester/service.rb

```diff
@@ -81,6 +90,18 @@ def _load_resources(version_module)
       end
     end # Class methods
 
+    def logger
+      @_logger ||= Rester.logger
+    end
```

Comment:
I think this should be `@_logger || Rester.logger`.  My thinking is we don't want to cache the value of `Rester.logger` in case it changes. If it changes, it's supposed to be a global logger by default.  Only if `Service#logger=` is called should it be set to something other than `Rester.logger`.